### PR TITLE
Update copyright symbol for NuGet packages

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

Use the HTML code for copyright symbol to avoid encoding issues.

## PR Context

Sometimes encoding issues cause the copyright symbol to not work with nuget.org validations.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Comment by adityapatwardhan** [comment]
Validated the NuGet package from a private build.

**Review by anmenaga — APPROVED** [review]

File: tools/packaging/packaging.strings.psd1 [hunk 1]
```diff
@@ -127,7 +127,7 @@ open {0}
         <licenseUrl>https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt</licenseUrl>
         <tags>PowerShell</tags>
         <language>en-US</language>
-        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+        <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
```
Comment (iSazonov) [inline tools/packaging/packaging.strings.psd1]: Why not "(c)"? We use in all files:
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Comment (daxian-dbw) [reply tools/packaging/packaging.strings.psd1]: Agreed. Maybe we can change to use `(c)`?

Comment (adityapatwardhan) [reply tools/packaging/packaging.strings.psd1]: NuGet.org documentation needs it this way or the copyright symbol.

Comment (TravisEz13) [reply tools/packaging/packaging.strings.psd1]: agree, our legal teams says OSS should use (c).  I sent @adityapatwardhan 	 the link offline.

Comment (adityapatwardhan) [reply tools/packaging/packaging.strings.psd1]: I checked with nuget folks, and the only acceptable values are the copyright symbol and &#169;

**Pull request merged**